injectproxy/routes_test.go

```diff
@@ -671,18 +671,19 @@ func TestSeriesWithPost(t *testing.T) {
 
 func TestQuery(t *testing.T) {
 	for _, tc := range []struct {
-		name           string
-		labelv         string
-		staticLabelVal string
-		promQuery      string
-		promQueryBody  string
-		method         string
-
-		expCode          int
-		expPromQuery     string
-		expPromQueryBody string
-		expResponse      []byte
-		errorOnReplace   bool
+		name                 string
+		labelv               string
```

Comment:
what about using a string slice instead of `secondLabelv` and `forceAddSecondLabelv`?
```suggestion
		labelv               []string
```

Reply:
Yes, really elegant solution